src/components/Header.js

```diff
@@ -6,9 +6,16 @@ import HeaderBranding from "./HeaderBranding";
 import SearchBar from "./SearchBar";
 import DarkModeToggle from "./DarkModeToggle";
 
+const searchIndices = [
+  {
+    name: `Posts`,
+    title: `Posts`,
+  },
+];
+
 const Header = ({ isLandingPage = false }) => (
   <HeaderContainer isLandingPage={isLandingPage}>
-    <Link to={isLandingPage ? "/" : "/home"}>
+    <Link to={"/"}>
```

Comment:
Why are we removing this? There really isn't a reason to go back to the landing page